ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/ShopperInsightsClient.kt

```diff
@@ -189,19 +188,26 @@ class ShopperInsightsClient internal constructor(
     }
 
     /**
-     * Call this method when the PayPal button has been selected/tapped by the buyer.
+     * Call this method when the PayPal, Venmo or Other button has been successfully displayed to the buyer.
      * This method sends analytics to help improve the Shopper Insights feature experience.
+     * @param buttonType Type of button presented - PayPal, Venmo, or Other.
+     * @param presentmentDetails Detailed information, including button order, experiment type,
+     * and page type about the payment button that is sent to analytics to help improve the Shopper
+     * Insights feature experience.
      */
-    fun sendPayPalSelectedEvent() {
-        braintreeClient.sendAnalyticsEvent(PAYPAL_SELECTED, analyticsParams)
-    }
+    fun sendSelectedEvent(
+        buttonType: ButtonType,
+        presentmentDetails: PresentmentDetails
```

Comment:
I don't think we needed presentment details for this method, just the button type.